src/modules/emotes/style.css

```diff
@@ -1,33 +1,36 @@
 /* Prevent stacking of IceCube, SoSnowy */
-.bttv-emo-5849c9a4f52be01a7ee5f79d + .bttv-emo-5849c9a4f52be01a7ee5f79d,
-.bttv-emo-567b5b520e984428652809b6 + .bttv-emo-567b5b520e984428652809b6 {
+.bttv-global-emo-5849c9a4f52be01a7ee5f79d + .bttv-global-emo-5849c9a4f52be01a7ee5f79d,
```

Comment:
this looks like your branch is out of date with master. you may need to rebase off latest commit.